integration_test/testgenerated/server/server-service.yml

```diff
@@ -15,16 +19,16 @@ services:
       echo:
         http: GET /echo
         auth: "cookie:PALANTIR_TOKEN"
-      getPathParam:
-        http: GET /path/{myPathParam}
-        auth: header
-        args:
-          myPathParam: string
       getPathParamAlias:
         http: GET /path/alias/{myPathParam}
         auth: header
         args:
           myPathParam: StringAlias
+      getPathParam:
+        http: GET /path/string/{myPathParam}
```

Comment:
Whats the reason for adding in the `/string/` to the path?

Reply:
Yeah this is kind of lame. It turns out this was not working before, but we never actually registered the routes with httprouter and hit this panic when we do: https://github.com/julienschmidt/httprouter/blob/v1.3.0/tree.go#L163-L167

```
panic: 'alias' in new path '/path/alias/:myPathParam' conflicts with existing wildcard ':myPathParam' in existing prefix '/path/:myPathParam'
```

I'm not sure if this is a w-g-s bug and we should be handling these...

Reply:
This appears to be an issue specific to `julienschmidt/httprouter` since it uses prefix matching on the routes. Since there can only be 1 path to 1 route, it doesn't know which route to choose here since `alias/:name` could be matched for the same `/path/:name` portion. It seems like adding in the static path segment `string` resolves the common prefixes.